setup.cfg

```diff
@@ -1,7 +1,7 @@
 [aliases]
 test=pytest
 [tool:pytest]
-addopts=--verbose --cov-config .coveragerc --cov ring
+addopts=--verbose
```

Comment:
This line should not be changed.

Reply:
Fix it in 82cf5a980f4b8a23271d6cccac835cbf57b5212d